clang/lib/Parse/ParsePragma.cpp

```diff
@@ -401,6 +401,12 @@ struct PragmaMaxTokensTotalHandler : public PragmaHandler {
                     Token &FirstToken) override;
 };
 
+struct PragmaExportHandler : public PragmaHandler {
```

Comment:
This patch is big enough... I am beginning to wonder if we should split this pragma off into a separate patch.